validation/validation.py

```diff
@@ -118,7 +118,7 @@ def print_accuracy(self, MSF, symmetry, higher_order=True, use_fmm=False):
         print_info([MSF], [len(exc.get_electrostatic_active_elements()[0])], [duration], [correct], [computed], [err])
         return duration, err
      
-    def args_to_symmetry(args):
+    def args_to_symmetry(self, args):
```

Comment:
"PEP8 states that self should always be used: Always use self for the first argument to instance methods."

Reply:
It wasn't an instance method before, but a static method (see https://stackoverflow.com/questions/735975/static-methods-in-python). Looking back I should've added `@staticmethod`. I will approve since it's a trivial thing